src/main/java/uk/gov/hmcts/reform/managecase/api/controller/NoticeOfChangeController.java

```diff
@@ -298,7 +302,7 @@ public RequestNoticeOfChangeResponse requestNoticeOfChange(@Valid @RequestBody R
             message = AuthError.UNAUTHORISED_S2S_SERVICE
         )
     })
-    public SubmitCallbackResponse checkNoticeOfChangeApproval(@Valid @RequestBody CheckNoticeOfChangeApprovalRequest
+    public SubmitCallbackResponse checkNoticeOfChangeApproval(@Valid @RequestBody NoticeOfChangeRequest
```

Comment:
~~Why the class name change from ACA-85.  Should this not be renamed to `SubmitCallbackRequest` in a similar fashion to your new AboutToSubmitCallbackRequest/AboutToSubmitCallbackResponse.  If changing change in ACA-85 branch in pull through and merge carefully.~~

IF the two callback APIs share the same request structure (as per https://tools.hmcts.net/confluence/display/RCCD/Callbacks+%28Webhooks%29+documentation) then the should share the same class with a more generic name.